templates/engage/shared/_header-webinar.html

```diff
@@ -0,0 +1,23 @@
+<section class="{% if class %}p-strip {{ class }}{% else %}p-strip--light{% endif %}">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="{% if image %}col-7{% else %}col-8{% endif %}">
+      <h1 style="font-weight: 100;">
```

Comment:
Maybe a bespoke styling to remove the margin bottom of this is the only-child. Save a lot of null checking logic.